src/smolagents/models.py

```diff
@@ -469,6 +465,7 @@ def __init__(
         if device_map is None:
             device_map = "cuda" if torch.cuda.is_available() else "cpu"
         logger.info(f"Using device: {device_map}")
+        self.is_vlm = False
```

Comment:
As this attribute is private (the users are not expected to set/modify it), maybe we could name it: `self._is_vlm`.